config/initializers/pry_warning.rb

```diff
@@ -0,0 +1,3 @@
+if ENV['PRY_WARNING'] && defined? pry
```

Comment:
> Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

Rebased onto develop and you better not report this one, or else... ;-)

-- 
Later,
 Lukas #lzap Zapletal